### PR TITLE
Print network and time info when the board starts

### PR DESCRIPTION
**Pull request by ferrarimarco**
Printing network info and the current time (when setting that up) is useful for debugging purposes. Due to the "example" nature of this project I think it's useful to show this when a user is moving first steps with a ESP32 board.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -38,14 +38,14 @@ def connect():
         sta_if.connect(config.wifi_config['ssid'], config.wifi_config['password'])
         while not sta_if.isconnected():
             pass
-    #print('network config:', sta_if.ifconfig())
+    print('network config:', sta_if.ifconfig())
```
Comment (gguuss) [inline main.py]: Prefer string format over concatenation:

```python
print('network config: {}'.format(sta_if.ifconfig()))
```

File: main.py [hunk 2]
```diff
@@ -38,14 +38,14 @@ def connect():
...
 
 def set_time():
     ntptime.settime()
     tm = utime.localtime()
     tm = tm[0:3] + (0,) + tm[3:6] + (0,)
     machine.RTC().datetime(tm)
-    #print(utime.localtime())
+    print('current time:', utime.localtime())
```
Comment (gguuss) [inline main.py]:
```python
print('current time: {}'.format(utime.localtime()))
```

**Review by gguuss — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (ferrarimarco) [reply main.py]: done

File: main.py (hunk 2 above)
Comment (ferrarimarco) [reply main.py]: done

**Comment by ferrarimarco** [comment]
I've addressed your comments and rebased on the latest master

**Comment by ferrarimarco** [comment]
Thx @gguuss !

**Pull request merged**